word_scripts/word_file_merger.py

```diff
@@ -0,0 +1,19 @@
+
+path_to_save = '../app/src/main/assets/words/'
+file_prefix = 'words-length-'
+
+def word_merger(dict_of_words):
+    for length, words in dict_of_words.items():
+        with open('{0}{1}{2}'.format(path_to_save, file_prefix, length), 'r') as file:
+            wordsByLength = file.readlines()
+        
+        for word in words:
+            if '{0}\n'.format(word) in wordsByLength:
```

Comment:
Right now this is `O(n^2)` because python checks if an element is in a list by iterating through each element. Use a `set` for linear runtime.